Binary files orig/uFTIR/src/.Makevars.win.swp and patched/uFTIR/src/.Makevars.win.swp differ
diff -Nru orig/uFTIR/src/Makevars.win patched/uFTIR/src/Makevars.win
--- orig/uFTIR/src/Makevars.win	2020-05-11 14:17:50.000000000 +0200
+++ patched/uFTIR/src/Makevars.win	2020-07-02 18:06:47.223878900 +0200
@@ -1,16 +1,17 @@
-VERSION = 2.2.3
-COMPILED_BY ?= gcc-4.6.3
-RWINLIB = ../windows/gdal2-$(VERSION)
-TARGET = lib$(subst gcc,,$(COMPILED_BY))$(R_ARCH)
-
 PKG_CPPFLAGS =\
-	-I$(RWINLIB)/include/gdal \
-	-I$(RWINLIB)/include/geos \
-	-I$(RWINLIB)/include/proj
 
 PKG_LIBS = \
-	-L$(RWINLIB)/$(TARGET) \
-        -L$(RWINLIB)/lib$(R_ARCH) \
+        -lgdal -lgta -lsqlite3 -lmysqlclient -lspatialite -lproj -lgeos_c -lgeos  \
+        -ljson-c -lnetcdf -lpq \
+        -lwebp -lcurl -lidn2 -lunistring -lssh2 -lgcrypt -lgpg-error -lssl \
+        -lhdf5_hl -lhdf5 -lexpat -lfreexl \
+        -lmfhdf -ldf -lportablexdr  -lpcre \
+        -lopenjp2 -ljasper -lpng -ljpeg -ltiff -lwebp -lgeotiff -lgif -lxml2 -llzma \
+        -lodbc32 -lodbccp32 -liconv -lpsapi -lwldap32 -lsecur32 -lgdi32 -lnormaliz \
+        -lbz2 -lzstd -lz -lcrypto -lcrypt32 -lcfitsio -lws2_32
+
+
+PKG_LIBS0 = \
 	-lgdal -lsqlite3 -lspatialite -lproj -lgeos_c -lgeos  \
 	-ljson-c -lnetcdf -lmariadbclient -lpq -lintl -lwebp -lcurl -lssh2 -lssl -lcrypto \
 	-lkea -lhdf5_cpp -lhdf5_hl -lhdf5 -lexpat -lfreexl -lcfitsio \
@@ -18,17 +19,11 @@
 	-lopenjp2 -ljasper -lpng16 -ljpeg -ltiff -lgeotiff -lgif -lxml2 -llzma -lszip -lz \
 	-lodbc32 -lodbccp32 -liconv -lpsapi -lws2_32 -lcrypt32 -lwldap32 -lsecur32 -lgdi32
 
-all: clean winlibs
+all: clean 
 
 CXX_STD = CXX11
 
-winlibs:
-	mkdir -p ../inst
-	"${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" --vanilla "../tools/winlibs.R" $(VERSION)
-	cp -r $(RWINLIB)/share/gdal ../inst/
-	cp -r $(RWINLIB)/share/proj ../inst/
-
 clean:
 	rm -f $(SHLIB) $(OBJECTS)
 
-.PHONY: all winlibs clean
+.PHONY: all clean
